cli/providers/aws/credentials.go

```diff
@@ -6,10 +6,15 @@ import (
 	"github.com/pchico83/i2kit/cli/schemas/environment"
 )
 
-func getAWSConfig(e *environment.Environment) (*aws.Config, error) {
+//GetConfig returns a config aws object
+func GetConfig(p *environment.Provider) (*aws.Config, error) {
```

Comment:
I'd just pass accesskey and secretkey, and region instead of the full object.

Reply:
Let's pass a DNSProvider object